hmda/src/main/scala/hmda/validation/rules/lar/validity/V632_3.scala

```diff
@@ -23,7 +23,9 @@ object V632_3 extends EditCheck[LoanApplicationRegister] {
     val coEthnicity = lar.coApplicant.ethnicity
     when(coEthnicity.ethnicityObserved is equalTo(NotVisualOrSurnameEthnicity)) {
       (coEthnicity.ethnicity1 is containedIn(validEthnicityList)) or
-        (coEthnicity.otherHispanicOrLatino not empty)
+        (coEthnicity.ethnicity1 is equalTo(EmptyEthnicityValue) and
```

Comment:
This would not flag the edit. The issue has a sample file that would help. Edit is:
```
1) ethnicity of co-applicant or co-borrower must equal 1, 11, 12, 13, 14, 2, 3 or be blank
2) ethnicity of co-applicant or co-borrower can ONLY be blank if ethnicity of co-applicant or co-borrower: free form text field for other hispanic or latino is NOT blank/empty. 
```